tensorboard/plugins/npmi/http_api.md

```diff
@@ -90,3 +98,41 @@ inner array resolves to the metrics per annotation. Example:
   ]
 }
 ```
+
+## `data/plugin/npmi/embeddings`
+
+Retrieves high-dimensional embeddings for the annotations. Embeddings are
+optional but can be used to further inform the user in their analysis.
+
+Returns a dictionary mapping from `annotation` to an embedding. The embedding
```

Comment:
The implementation and tests suggest this endpoint's returned map actually keys on `runName` rather than `annotation`, and that the values are not single embeddings but rather the rank-2 tensors of all embeddings for the annotations for that run.  Assuming that's correct, could you update the docs and example here?